src/NumericalAlgorithms/Spectral/Chebyshev.cpp

```diff
@@ -119,3 +119,10 @@ std::pair<DataVector, DataVector> compute_collocation_points_and_weights<
 /// \endcond
 
 }  // namespace Spectral
+
+/*
```

Comment:
This won't show up in the documentation.  Perhaps put a citation in the hpp file, and don't use \cite in a non-doxygen comment

Reply:
Hm these Kopriva citations only appear in cpp files since they provide algorithm implementations. They don't even necessarily have an hpp. I thought adding this _References_ section to the bottom of the file where the citation appears and using the doxygen \cite for both for making clear that this is a citation and allowing for the comment to be moved somewhere else eventually made sense. If you have another suggestion on how to do things I can change it.